### PR TITLE
Update ingress apiVersion on  kubernetes.io/docs/tasks/access-applica…

### PR DESCRIPTION
**Pull request by RA489**
Update ingress apiVersion on  https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/

NOTE: networking.k8s.io/v1 is available in Kubernetes v1.19+

### CODE REVIEW

**Comment by tengqm** [comment]
No. We are not encouraging users to use deprecated API versions. Never.

**Review by sftim — COMMENTED** [review]
/sig network
/kind cleanup
/lgtm

**Comment by sftim** [comment]
This could use another LGTM, then (I'm PR wrangler this week) I can approve it.

**Comment by sftim** [comment]
Changed my mind
/lgtm cancel

**Comment by RA489** [comment]
Any review update on this PR?

File: content/en/examples/service/networking/example-ingress.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1
+apiVersion: networking.k8s.io/v1beta1
```
Comment (kbhawkey) [inline content/en/examples/service/networking/example-ingress.yaml:1]: Hi @RA489 .
Does it make sense to update this example?
Based on:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#ingress-v1-networking-k8s-io
https://kubernetes.io/docs/reference/kubernetes-api/services-resources/ingress-v1/#Ingress

Comment (RA489) [reply content/en/examples/service/networking/example-ingress.yaml:1]: Thanks for the review. I think this PR no longer needs to be updated.

**Pull request closed without merging**